package.json

```diff
@@ -8,6 +8,8 @@
   "author": "James Long",
   "license": "MIT",
   "main": "./index.js",
+  "browser": "./standalone.js",
+  "unpkg": "./standalone.js",
```

Comment:
How about move `unpkg` field here

https://github.com/prettier/prettier/blob/4b95c408ae67820ba450e5f409139ccb471ff268/scripts/build/build.js#L80

Only bundled `standalone.js` can import from unpkg.com

Reply:
But nobody's going to publish an unbundled version on unpkg. Or am I missing something?

Reply:
No, never mind.